backstop/tests/select.html

```diff
@@ -38,8 +39,10 @@
         height: 210px;
       }
 
-      #demo-inline-combo {
-        flex-direction: row;
+      #demo-in-context {
```

Comment:
Remove this... too much custom logic on simple demo and lets merge this..

Reply:
Reverted back to flex.